### ISSUE TITLE
Build failed on ResolveSdks

### ISSUE THREAD

**Issue by nielscil** [xamarin/xamarin-android#2907]
### Steps to Reproduce

1. Create new xamarin android project.
2. Try Building it.
3. Failed on ResolveSdks task (see log for the exception).

### Expected Behavior

Build of project should succeed.

### Actual Behavior

Build failed on ResolveSdks

### Version Information

Microsoft Visual Studio Professional 2017 
Version 15.9.6
VisualStudio.15.Release/15.9.6+28307.344
Microsoft .NET Framework
Version 4.7.03056

Installed Version: Professional

Xamarin   4.12.3.79 (d15-9@260fa6a34)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   4.16.13 (45a16efd4)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   1.1.128 (6f5ebb2)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   9.1.5.0 (HEAD/4b951a3e7)
Xamarin.Android Reference Assemblies and MSBuild support.

Xamarin.iOS and Xamarin.Mac SDK   12.2.1.12 (65ec520)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.

### Log File

`The "ResolveSdks" task failed unexpectedly.
System.IO.FileLoadException: Could not load file or assembly 'Xamarin.Android.Tools.AndroidSdk, Version=1.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
File name: 'Xamarin.Android.Tools.AndroidSdk, Version=1.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065' ---> System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
The Zone of the assembly that failed was:
MyComputer
   at Xamarin.Android.Tasks.ResolveSdks.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()`

**Comment by pjcollins** [comment]
I haven't managed to reproduce this anywhere yet. Would you be able to try the recent [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) release to see if the issue that you are encountering still reproduces there? I am wondering if this could somehow be caused by some sort of upgrade/installation issue. If VS 2019 is working for you, perhaps a full re-install of VS2017 will resolve the issue as well?

**Comment by nielscil** [comment]
I've tried Visual studio 2019 preview and there it works well. Unfortunately it gave other problems when trying to compile our normal app. I already tried a re-install of VS2017 but it didn't work either. I've solved the issue at this moment setting `AllowStrongNameBypass` in the register to `1` [(How to: Disable the Strong-Name Bypass Feature)](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/how-to-disable-the-strong-name-bypass-feature).
